GoogleChromeLabs/wp-sitemaps
# Update WPCS to v2.2

### Description
This updates WPCS to version 2.2. See release notes: https://github.com/WordPress/WordPress-Coding-Standards/releases/tag/2.2.0

### Type of change
Please select the relevant options:


### Steps to test
Checkout the branch, run `composer update` and see that the sniffs are still working.


### Acceptance criteria
